src/main/scala/beam/utils/beamToVia/appsForVisualizations/visualization_11.scala

```diff
@@ -0,0 +1,71 @@
+package beam.utils.beamToVia.appsForVisualizations
```

Comment:
Does it make sense to attach some document/actual vidoe what does visualization 11 and others mean?

Reply:
a good question ..